doc/spec_TransferElement.dox

```diff
@@ -138,48 +138,53 @@ This documnent is currently still **INCOMPLETE**!
+    - \anchor transferElement_B_9_1_3 9.1.3 TransferElement::setExceptionBackend() is virtual. Decorator-like TransferElements overload it so all target elements also get the correct exception backend.
+    - \anchor transferElement_B_9_1_4 9.1.4 Decorators set their TransferElement::_exceptionBackend in the constructor to their target's _exceptionBackend. \ref transferElement_comment_B_9_1_4 "(*)"
+    - 9.1.5 After replacing a low level transfer element in TransferElement::replaceTransferElement(), the exception backend of the new transfer element must be adapted.
+  - \anchor transferElement_B_9_2 9.2 DeviceBackend::setException() triggers the actions described in \ref transferElement_B_9_3 "9.3", \ref transferElement_B_9_4 "9.4", \ref transferElement_B_9_5 "9.5"  and \ref transferElement_B_9_6 "9.6" [\ref UnifiedTest_TransferElement_B_9_2 "U" (with all sub-points)]
+    - \anchor transferElement_B_9_2_1 9.2.1 Like for the activation of the asynchronous read (cf. \ref transferElement_B_8_5_7 "8.5.7"), each backend is only responsible for
+       take the actions in DeviceBackend::setException() for those TransferElements it created.
+       Meta-backends like the LogicalNameMappingBackend delegate this call to all of their target backends, as they also delegated the creation.
+    - 9.2.2 Further calls to DeviceBackend::setException() have no effect until Device::open() has successfully been called.
+    - \anchor transferElement_B_9_2_3 9.2.3 Application code can call Device::setException() to trigger the exception state \ref transferElement_comment_B_9_2_3 "(*)"
+  - \anchor transferElement_B_9_3 9.3 TransferElements with wait_for_new_data flag
+    - \anchor transferElement_B_9_3_1 9.3.1 Each transfer element deactivates asynchronous reads so no further data is pushed into the \ref transferElement_data_transport_queue "implementation-specific data transport queue" until open() has been called successfully in the backend (*) and Device::activateAsyncRead() has been called [\ref UnifiedTest_TransferElement_B_9_3_1_single "U"]
+    - \anchor transferElement_B_9_3_2 9.3.2 Exactly one ChimeraTK::runtime_error is pushed into the queue of *each* transfer elements of the backend with wait_for_new_data (*). This must happen after 9.3.1. to avoid race conditions. [\ref UnifiedTest_TransferElement_B_9_3_2_single "U"]
+    - 9.3.3 The first data on the queue after the exception is the initial value send when calling Device::activateAsyncRead() (see 8.5.2).
+    - 9.3.4 The backend must make sure 9.3.2 is finished before a call to Device::activateAsyncRead() re-activates the asynchronous transfers again (*).
+  - \anchor transferElement_B_9_4 9.4 TransferElements without wait_for_new_data [\ref UnifiedTest_TransferElement_B_9_4 "U"]
+     - \anchor transferElement_B_9_4_1 9.4.1 Each call to doReadTransferSynchonously() will throw a ChimeraTK::runtime error until open() has been called successfully.
```

Comment:
then throw->throws

Reply:
I will put it in. But it really goes a bit against my language feeling. When setException is executed, the future call to doReadTransferSynchonously() has not happened yet. "When in future it is called, then it will throw." That's correct english. "When in future it is called, then it throws." is somewhat fishy.